### PR TITLE
#25 wrote some integration tests for routes

### PR DESCRIPTION
**Pull request by SailReal**
see #25

### CODE REVIEW

File: test/FunctionalSpec.scala [hunk 1]
```diff
@@ -1,41 +1,201 @@
...
+                  .withAuthenticator[DefaultEnv](identity.loginInfo))
+                )
+
+                status(result) must beEqualTo(NO_CONTENT)
+            }
         }
 
-        "send 200 on a good request" in {
-            route(app, FakeRequest(GET, "/signIn")).map(status(_)) mustBe Some(OK)
+        "return ok on `onAddPlayer`" in new Context {
```
Comment (DonatJR) [inline test/FunctionalSpec.scala]: this is a duplicate from line 33 or am I not seeing this correctly?

**Review by DonatJR — COMMENTED** [review]
is there something preventing us from testing the websocket and logup/login routes as well?
also, we should probably remove the onExit route (we don't use it anyway right now and this would increase our coverage)

**Comment by SailReal** [comment]
Login/Logout is possible, I will implement that...tried testing actors with https://doc.akka.io/docs/akka/2.5.5/scala/testing.html but didn't find a working solution right now so testing the websocket we have to investigate more time.

Yes, should I remove the onExit?

**Comment by DonatJR** [comment]
sorry for not reading all of the testing page, but wouldn't it be enough to call the `createWebSocket` route and see if a instance of a certain class is returned? we don't have to test the actor itself (for now), just the controller. If I'm not getting it just leave it as is and we will figure it out later.

:+1: for removing onExit

**Review by DonatJR — APPROVED** [review]

**Pull request merged**